xml/telnet_banners.xml

```diff
@@ -47,6 +47,8 @@
     <param pos="0" name="service.vendor" value="Cisco"/>
     <param pos="0" name="os.vendor" value="Cisco"/>
     <param pos="0" name="hw.vendor" value="Cisco"/>
+    <param pos="0" name="os.family" value="IOS"/>
+    <param pos="0" name="os.product" value="IOS"/>
```

Comment:
IIRC this can match on Cisco switches as well which means the OS can be CAT OS for Cisco Catalyst gear.

Reply:
👍.  We've definitely run into this before, and if it can be either, I think the pattern is to leave off product/family.